lib/node_modules/@stdlib/math/base/tools/hermitepoly/test/test.factory.js

```diff
@@ -0,0 +1,531 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2018 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var EPS = require( '@stdlib/constants/math/float64-eps' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var factory = require( './../lib' ).factory;
+
+
+// FIXTURES //
+
+var random2 = require( './fixtures/python/random2.json' );
+
+var mediumNegative1 = require( './fixtures/python/medium_negative_1.json' );
+var mediumNegative2 = require( './fixtures/python/medium_negative_2.json' );
+var mediumNegative5 = require( './fixtures/python/medium_negative_5.json' );
+
+var mediumPositive1 = require( './fixtures/python/medium_positive_1.json' );
+var mediumPositive2 = require( './fixtures/python/medium_positive_2.json' );
+var mediumPositive5 = require( './fixtures/python/medium_positive_5.json' );
+
+var smallPositive1 = require( './fixtures/python/small_positive_1.json' );
+var smallPositive2 = require( './fixtures/python/small_positive_2.json' );
+var smallPositive5 = require( './fixtures/python/small_positive_5.json' );
+
+var smallNegative1 = require( './fixtures/python/small_negative_1.json' );
+var smallNegative2 = require( './fixtures/python/small_negative_2.json' );
+var smallNegative5 = require( './fixtures/python/small_negative_5.json' );
+
+var tiny1 = require( './fixtures/python/tiny_1.json' );
+var tiny2 = require( './fixtures/python/tiny_2.json' );
+var tiny5 = require( './fixtures/python/tiny_5.json' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof factory, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function returns a function', function test( t ) {
+	t.strictEqual( typeof factory( 2 ), 'function', 'returns a function' );
+	t.end();
+});
+
+tape( 'the function returns a function which accurately computes `Hen(x)` for random `n` and `x`', function test( t ) {
```

Comment:
Since you removed the `e` subscript from the README equation, should we remove the subscript in the test descriptions?

Reply:
Absolutely, nice catch.  I'm removing the ```e``` subscript base on the wikipedia page.

Reply:
Right now I'm using ```H_n```, but maybe we should forget the underscore?  I suppose it's a matter of taste.